src/js/directives/autoHideShow.js

```diff
@@ -0,0 +1,147 @@
+/** @module AutoHideShowControls
+* Directive used to autohide the UI. Can be instantiated by setting data-mstr-directive="autoHideShow" on an element.
+*/
+
+let TIME_TILL_FADE = 3000;
```

Comment:
It can be configured, see l. 19;
```
        if (Number.isFinite(this.config.timeToFade)) {
            TIME_TILL_FADE = this.config.timeToFade * 1000;
        }
```